src/client/user/components/UserLinkTable/StatusBar/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { Typography, createStyles, makeStyles } from '@material-ui/core'
+import { Alert, AlertTitle } from '@material-ui/lab'
+import StatusBarCompletedIcon from '../../../widgets/StatusBarCompletedIcon'
+import StatusBarFailedIcon from '../../../widgets/StatusBarFailedIcon'
+import StatusBarInProgressIcon from '../../../widgets/StatusBarInProgressIcon'
+import userActions from '../../../actions'
+import useAppMargins from '../../../../app/components/AppMargins/appMargins'
+import { StatusBarVariant } from '../../../reducers/types'
+import { GoGovReduxState } from '../../../../app/reducers/types'
+
+type StyleProps = {
+  appMargins: number
+}
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      alignItems: 'stretch',
+    },
+    error: {
+      backgroundColor: '#FFEDED',
+      color: theme.palette.text.primary,
+    },
+    info: {
+      backgroundColor: '#FDF3E8',
+      color: theme.palette.text.primary,
+    },
+    success: {
+      backgroundColor: '#EAF9E7',
+      color: theme.palette.text.primary,
+    },
+    content: {
+      paddingTop: theme.spacing(2),
+      paddingBottom: theme.spacing(2),
+      [theme.breakpoints.up('md')]: {
+        paddingRight: (props: StyleProps) => props.appMargins,
+        paddingLeft: (props: StyleProps) => props.appMargins,
+      },
+      [theme.breakpoints.down('sm')]: {},
+    },
+    message: {},
```

Comment:
```suggestion
```

Reply:
thought to leave it there for ease of styling next time, what do you think?

Reply:
sure, would be good too!